datasets/ted_talks_iwslt_2/README.md

```diff
@@ -0,0 +1,321 @@
+
+### Languages
+
+Ted talks are mostly held in English (`en`). Almost all of the talks have been translated, by volunteers, into Arabic, Bulgarian, Chinese (simplified), French, Italian, Korean, Portuguese (Brazil) and Spanish. For about 70 other languages, the number of translated talks ranges from several hundreds (e.g. such as other Dutch, German, Hebrew, Romanian) to one (e.g. Hausa, Hupa, Bislama, Ingush, Maltese)
+
+## Dataset Structure
+
+### Data Instances
+
+```
+<file id="1">
+  <head>
+    <url>http://www.ted.com/talks/ryan_holladay_to_hear_this_music_you_have_to_be_there_literally.html</url>
+    <pagesize>66634</pagesize>
+    <dtime>Sun Jan 12 15:17:32 CET 2014</dtime>
+    <content-type>text/html; charset=utf-8</content-type>
+    <encoding>utf-8</encoding>
+    <videourl>http://download.ted.com/talks/RyanHolladay_2013S.mp4</videourl>
+    <videopath>talks/RyanHolladay_2013S.mp4</videopath>
+    <transcription>
+      <seekvideo id="2939">(Music)</seekvideo>
+      <seekvideo id="7555">For any of you who have visited or lived in New York City,</seekvideo>
+      <seekvideo id="11221">these shots might start to look familiar.</seekvideo>
+      <seekvideo id="16116">This is Central Park,</seekvideo>
+      .
+      .
+      .
+      <seekvideo id="361992">for people to interact with</seekvideo>
+      <seekvideo id="363709">and experience music.</seekvideo>
+      <seekvideo id="365451">Thank you.</seekvideo>
+      <seekvideo id="367495">(Applause)</seekvideo>
+    </transcription>
+    <talkid>1903</talkid>
+    <title>Ryan Holladay: To hear this music you have to be there. Literally</title>
+    <description>The music industry ......segments of sounds that only play when a listener is physically nearby. (Filmed at TED@BCG.)</description>
+    <keywords>entertainment,music,technology</keywords>
+    <image>http://images.ted.com/images/ted/d98c17773da6f84e9f915895c270c7ffd2de3778_389x292.jpg</image>
+    <date>2014/01/12</date>
+    <wordnum>885</wordnum>
+    <charnum>5051</charnum>
+  </head>
+  <content>(Music) For any of you who have visited or lived in New York City, these shots might start to look familiar. This is Central Park, ............new ways for people to interact with and experience music. Thank you. (Applause)</content>
+</file>
+```
+
+### Data Fields
+
+For each language, a single XML file is generated which includes all talks subtitled in 
+that language. Each talk is enclosed in tags `<file id="int">` and `</file>` and includes, among other tags: 
+
+| Tags | Description |
+|---|:---|
+| `<url>`| the address of the original HTML document of the talk |
+| `<speaker>` | the name of the talk speaker |
+| `<talkid>` | the numeric talk identifier |
+| `<transcript>` | talk subtitles split in captions |
+| `<date>` | the issue date of the talk |
+| `<content>` |  talk subtitles |
+
+
+### Data Splits
+
+Dat has been split based on the year
+2014 -> Train
+2015 -> Dev
+2016 -> Test
```

Comment:
the paper doesn't suggest any splits. So did my own _arbit_ splits. Any suggestions?

Reply:
Maybe the year can be a parameter as well ?
```
load_dataset("ted_talks_iwslt_2", lang1="fr", lang2="en", year="2015")
```
If `year` is not specified it can download all the years

Reply:
okay... I guess then there are no train/test/dev splits ?

Reply:
Yes in general we let the users define their splits if there's no split explicitly given by the authors.

For example users can use the `.train_test_split()` method